[b-6]: get movie by id

Jokeroff opened the pull request:
- enrichment reworked
- Dto for User, Review, Movie(extended)
- get movie by id added

Jokeroff committed:
[b-6]:
- enrichment reworked
- Dto for User, Review, Movie(extended)
- get movie by id added

Ramoid commented on src/main/java/com/lebediev/movieland/service/enrich/MovieEnrichmentService.java:
```diff
@@ -25,51 +28,92 @@
+    public void enrichMovieByGenres(Movie movie) {
+        log.info("Start enriching movie by genres ");
+        long startTime = System.currentTimeMillis();
+        List <Integer> params = Arrays.asList(movie.getMovieId());
```
На ворнинг решил забить))

Jokeroff replied:
быстрое решение )

Jokeroff committed:
[b-6]:
- User without password and email
- UserDto removed
- dependency for Appache datasource (to be changed)
